### PR TITLE
Add .NET Core 3.1 Support Warning

### PR DESCRIPTION
**Pull request by luisquintanilla**
## Summary

Include warning for users letting them know .NET for Spark currently targets out of support .NET Core 3.1.

[Preview Link](https://review.learn.microsoft.com/en-us/dotnet/spark/what-is-apache-spark-dotnet?branch=pr-en-us-33061)

### CODE REVIEW

File: includes/net-core-31-spark.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+> [!WARNING]
+> .NET for Apache Spark targets an out of support version of .NET (.NET Core 3.1). For more details see the [.NET Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core).
```
Comment (adegeo) [inline includes/net-core-31-spark.md:2]: Fixes the lint error

```suggestion
> .NET for Apache Spark targets an out of support version of .NET (.NET Core 3.1). For more details see the [.NET Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core).

```

**Review by adegeo — APPROVED** [review]
LGTM. Just so you know, you can create an **includes** folder at **docs/spark/includes** to help organize your content.

File: docs/spark/how-to-guides/connect-to-azure-storage.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ ms.custom: mvc,how-to
 
 In this article, you learn how to connect to an Azure Data Lake Storage (ADLS) Gen 2 or Windows Azure Storage Blob (WASB) account through an instance of [.NET for Apache Spark](https://github.com/dotnet/spark) running locally on your Windows machine.
 
-[!INCLUDE [.NET Core 3.1 Warning](../../includes/net-core-31-spark.md)]
+[!INCLUDE [.NET Core 3.1 Warning](./includes/net-core-31-spark.md)]
```
Comment (adegeo) [inline docs/spark/how-to-guides/connect-to-azure-storage.md:15]: You still need to go up one folder.

Comment (luisquintanilla) [reply docs/spark/how-to-guides/connect-to-azure-storage.md]: Good catch. I think I got it right this time...

**Pull request merged**